### PR TITLE
Full screen support

### PR DESCRIPTION
**Pull request by John-Theo**
Entering presentation page will trigger force fullscreen automatically, while exiting full screen will return back to upload page.

### CODE REVIEW

**Comment by John-Theo** [comment]
`.idea` files are committed, which is unintended...

**Comment by John-Theo** [comment]
@awmleer  waiting to be merged

File: src/pages/presentation.page.tsx [hunk 1]
```diff
@@ -230,9 +230,22 @@ export const PresentationPage: FC<RouteComponentProps> = (props) => {
       // togglePause()
     })
   })
+
+  const containerRef = useRef()
+  function GoInFullscreen(el: HTMLElement) {
```
Comment (awmleer) [inline src/pages/presentation.page.tsx]: Function names should be in small camel style.

File: src/pages/presentation.page.tsx [hunk 2]
```diff
@@ -230,9 +230,22 @@ export const PresentationPage: FC<RouteComponentProps> = (props) => {
...
+    el && el.requestFullscreen().then()
+  }
+  function onFullScreenChange () {
+    if (!!document.fullscreenElement === false) props.history.push('/upload')
```
Comment (awmleer) [inline src/pages/presentation.page.tsx]: It may not be suitable to go upload page when exiting full screen mode.

**Pull request merged**